### PR TITLE
EPMDJ-6744: Implement header

### PR DESCRIPTION
**Pull request by RomanAchilov**

### CODE REVIEW

File: src/css/custom.scss [hunk 1]
```diff
@@ -18,3 +18,8 @@
   --ifm-color-primary-lightest: rgb(146, 224, 208);
   --ifm-code-font-size: 95%;
 }
+
+.react-toggle {
```
Comment (IliaSolovev) [inline src/css/custom.scss]: Please, instead it paste `colorMode: {
      defaultMode: 'light',
      disableSwitch: true,
    },` in themeConfig

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/css/button.scss [hunk 2]
```diff
@@ -1,15 +1,17 @@
...
-@import "fonts";
+@import 'colors';
+@import 'fonts';
 
 .button-primary {
   display: flex;
   align-items: center;
   gap: 8px;
-  padding: 12px 48px;
+  padding: 12px 48px !important;
```
Comment (RomanDavlyatshin) [inline src/css/button.scss:8]: What is the reasoning for the usage of `!important`?

File: src/css/overrides.scss [hunk 3]
```diff
@@ -1,5 +1,41 @@
+%media {
```
Comment (RomanDavlyatshin) [inline src/css/overrides.scss]: I got two issues with that 😢 :

## The naming

The placeholder name `%media` is very vague 😞 (it's unclear which properties it affects or what it does).
A more descriptive name could fix that (e.g. `%max-width-breakpoints` or `%media-max-width`)

## The overall idea
I've looked into tailwind docs and I've found all the necessary customization options:
1. That one allows breakpoints customization https://tailwindcss.com/docs/breakpoints
2. That one allows container size & padding customization https://tailwindcss.com/docs/container

TL;DR I suggest customizing containers behavior with `tailwind.config.js`

**Review by RomanDavlyatshin — CHANGES_REQUESTED** [review]

**Comment by IliaSolovev** [comment]
As I understand you deleted `package-lock.json`. Pls return it

File: src/css/tailwind.css [hunk 4]
```diff
@@ -1,3 +1,3 @@
-@tailwind base;
 @tailwind components;
 @tailwind utilities;
+@tailwind base;
```
Comment (IliaSolovev) [inline src/css/tailwind.css:3]: Why you move it? And as I remember ww decide to remove base and components imports. Or you want to remove it in another PR?

File: src/theme/Footer/index.tsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+  { bg: 'bg-twiter', link: 'https://twitter.com/' },
+  { bg: 'bg-youtube', link: 'https://www.youtube.com/channel/UCJtegUnUHr0bO6icF1CYjKw/featured' },
+];
+
+const Footer = () => {
+  const { footer: { links } } = useThemeConfig();
+  return (
+    <footer className="bg-monochrome-light-tint">
+      <div className="container">
+        <section className="grid py-14 grid-cols-1 md:grid-cols-2 lg:grid-cols-footer" style={{ gap: '106px' }}>
```
Comment (IliaSolovev) [inline src/theme/Footer/index.tsx:24]: Are you sure that there should be `106px`? And maybe update tailwindConfig for it?

**Review by IliaSolovev — CHANGES_REQUESTED** [review]

File: src/css/_typography.scss [hunk 6]
```diff
@@ -49,6 +56,8 @@ em {
 
 //inline link
 a {
+  min-width: 24px; // for social icons without text
```
Comment (RomanDavlyatshin) [inline src/css/_typography.scss]: Remove that pls 😄

**Review by RomanDavlyatshin — APPROVED** [review]

**Review by IliaSolovev — APPROVED** [review]
